### PR TITLE
Fix/travis c ipushing

### PR DESCRIPTION
**Pull request by Chris-256**
- fix push command in script still being disabled
- change CI to only push with tags

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,18 +8,28 @@ jobs:
...
         - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
         - TAG=$(echo $TRAVIS_BRANCH | tr "/" "-")
 
         - bin/image build last4minors $TAG
         - bin/image test last4minors $TAG
-        - bin/image push last4minors $TAG
+
+
+    - name: Build and push docker images for the last 4 minor kube-versions
+      if: type = push AND branch = master AND tag IS present
```
Comment (tonyfahrion) [inline .travis.yml]: hmm, so cron wont update the images on docker hub?

Comment (Chris-256) [reply .travis.yml]: It does because cron triggers the first job not this one

**Comment by tonyfahrion** [comment]
good to go! 👍

**Comment by tonyfahrion** [comment]
@Chris-256 please add the tag

**Pull request merged**